Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM iojs:1.6
+FROM node:4
```

Comment:
Docker uses [4.2](https://github.com/docker-library/official-images/blob/master/library/node#L48). But this is not important, so if you want me to remove this, it's all fine to me.